src/cart/README.env

```diff
@@ -64,6 +64,11 @@ This file lists the environment variables used in CaRT.
    set ("D_LOG_MASK=DEBUG"), otherwise other priorities take precedence. The bit
    streams are passed as a parameter to D_DEBUG(mask, fmt, ...).
 
+ . D_LOG_FLUSH
+   Specifies priority level for flushing of messages into the log file.
+   Can be set to one of the following "DBUG,INFO,NOTE,WARN,ERR,CRIT,ALRT,EMRG,EMIT"
```

Comment:
DEBUG is also supported

Reply:
yeah we also support all lower-case or mixed case variations; not sure it is worth listing all of them